TI/Home.html

```diff
@@ -0,0 +1,107 @@
+          <ul class="navbar-nav me-auto mb-2 mb-lg-0">
+              <li class="nav-item">
+                  <a class="nav-link active" aria-current="page" href="About.html">About</a>
+              </li>
+              <li class="nav-item">
+                  <a class="nav-link" href="Projects.html">Projects</a>
+              </li>
+              <li class="nav-item">
+                  <a class="nav-link" href="Contact.html">Contact</a>
+              </li>
+          </ul>
+      </div>
+  </div>
+</nav>
+</header>
+<body>
+    <div class="container mt-3">
+        <h1 class="centered">Ismael Gauthier</h1>
+        <p class="centered">Développeur Web</p>
+    </div>
+        <div class="card mx-auto mx-b-10" style="width: 46rem;">
+            <img class="card-img-top" src="./assets/maTronche2.jpg" alt="Card image cap">
+            <div class="card-body">
+              <p class="card-text">Ma grosse tronche dans ton écran.</p>
+              <a href="About.html" class="btn btn-primary">A propos de ce mec!</a>
+            </div>
+          </div>
+    <div class="container">
+        <div class="row">
+            <div class="col-4">
+                <div class="card">
+                    <img src="./assets/a2.png" class="card-img-top" alt="...">
+                    <div class="card-body">
+                        <h5 class="card-title">Jeu de mémoire</h5>
+                        <p class="card-text">Mon projet de programmation 3</p>
+                        <a href="Projects.html" class="btn btn-primary">Accéder au projet</a>
+                    </div>
+                </div>
+            </div>
+            <div class="col-4">
+                <div class="card">
+                    <img src="./assets/i3.png" class="card-img-top" alt="...">
+                    <div class="card-body">
+                        <h5 class="card-title">IMDB</h5>
+                        <p class="card-text">Mon projet de techniques d'intégration 4</p>
+                        <a href="Projects.html" class="btn btn-primary">Accéder au projet</a>
+                    </div>
+                </div>
+            </div>
+            <div class="col-4">
+                <div class="card">
+                    <img src="./assets/f3.png" class="card-img-top" alt="...">
+                    <div class="card-body">
+                        <h5 class="card-title">Formulaire</h5>
+                        <p class="card-text">Mon projet de techniques d'intégration 3</p>
+                        <a href="Projects.html" class="btn btn-primary">Accéder au projet</a>
+                    </div>
+                </div>
+            </div>
+        </div>
+    </div>
+</body>
+<footer class="bg-light">
+    <div class="container py-3">
+        <div class="row">
+            <div class="col-md-6">
+                <a class="nav-link" href="https://www.facebook.com/profile.php?id=100016978482689">Facebook</a>

```

Comment:
Usually, external links must use target='_blank'. This way, the external page will be opened in a new tab, not replacing your current page.